### DISCUSSION TITLE
Two-factor authentication (No SMS area code of Vietnam)

### DISCUSSION THREAD

**Discussion by chaudev** [community/community#32711] [Other Feature Feedback, Questions, & Ideas]
I'm in Vietnam, I'm looking to enable two-factor authentication by SMS.
I can't find the country Vietnam and it's (+84) under "Contry Code".
I would like to know if Vietnam has a problem with Github or is this merely a shortcoming?
Is there another way for me to use this feature?
![Screen Shot 2022-09-12 at 11 17 34](https://user-images.githubusercontent.com/73950683/189573067-f667c07d-5692-43c8-aab9-465a64db8fa4.png)

**Comment by flyingangel** [comment]
Hello, I would like this feature as well. What is it needed for Vietnam to be supported? Just tell me maybe I can tell the government to allow access.
